tests/unit/transformers/test_categorical.py

```diff
@@ -359,38 +359,26 @@ def test___repr___default(self):
         assert stringified_transformer == 'OrderedUniformEncoder(order=<CUSTOM>)'
 
     def test__fit(self):
-        """Test the ``_fit`` method.
-
-        Validate that a unique integer representation for each category of the data is stored
-        in the ``categories_to_values`` attribute, and the reverse is stored in the
-        ``values_to_categories`` attribute. The order should match the ``self.order`` indices.
-        Setup:
-            - create an instance of the ``CustomLabelEncoder``.
-        Input:
-            - a pandas series.
-        Side effects:
-            - set the ``values_to_categories`` dictionary to the appropriate value.
-            - set ``categories_to_values`` dictionary to the appropriate value.
-        """
+        """Test the ``_fit`` method."""
         # Setup
-        data = pd.Series([1, 2, 3, 2, np.nan, 1, 1])
-        transformer = OrderedUniformEncoder(order=[2, 3, np.nan, 1])
+        data = pd.Series([1, 2, 3, 2, None, 1, 1])
```

Comment:
Why did you change this to None instead of np.nan?

Reply:
Yes that was not necessary at the end haha good catch, reverted in [df0ad68](https://github.com/sdv-dev/RDT/pull/681/commits/df0ad68efa2fd0b78ff118934e25ef03f8a3a926)